### PR TITLE
K8SPS-103: Add printer columns for orchestrator and router status

### PR DESCRIPTION
**Pull request by inelpandzic**
https://jira.percona.com/browse/K8SPS-103

### CODE REVIEW

File: deploy/operator.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ spec:
             fieldRef:
               apiVersion: v1
               fieldPath: metadata.namespace
-        image: perconalab/percona-server-mysql-operator:main
+        image: perconalab/percona-server-mysql-operator:k8sps-103-add-printer-columns
```
Comment (hors) [inline deploy/operator.yaml:44]: please remove all your changes connected with local tests

Comment (inelpandzic) [reply deploy/operator.yaml:44]: Yes, I was thinking about it. Should I remove them by hand or some script will do it?

Comment (inelpandzic) [reply deploy/operator.yaml:44]: Yes, I was thinking about it. Should I remove them by hand or some script will do it?

Comment (hors) [reply deploy/operator.yaml:44]: You should not add it when you push PR, your code will be tested correctly without these changes.

Comment (inelpandzic) [reply deploy/operator.yaml:44]: Should I generate CRDs? Is there some specific make command(s) I should run?

Comment (inelpandzic) [reply deploy/operator.yaml]: Fixed this with the help of @tplavcic and @egegunes .

**Comment by inelpandzic** [comment]
This is how it looks in the end:
```
➜  percona-server-mysql-operator git:(K8SPS-103-add-printer-columns) k get perconaservermysqls.ps.percona.com
NAME       REPLICATION   ENDPOINT                         STATE   ORCHESTRATOR   ROUTER   AGE
cluster1   async         cluster1-mysql-primary.default   ready   ready                   27m
```

I'm just not sure why the `Router` has no value.

**Comment by hors** [comment]
> This is how it looks in the end:
> 
> ```
> ➜   k get perconaservermysqls.ps.percona.com
> NAME       REPLICATION   ENDPOINT                         STATE   ORCHESTRATOR   ROUTER   AGE
> cluster1   async         cluster1-mysql-primary.default   ready   ready                   27m
> ```
> 
> I'm just not sure why the `Router` has no value. Do you maybe know @hors or @tplavcic ?

When you deploy the cluster you need to set 'group-replication' for 'spec.mysql.clusterType'  option and you will have GR instead of async and MySQL router will be present.

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-116/1a23a7a/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-116/1a23a7a/config.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-116/1a23a7a/init-deploy.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-116/1a23a7a/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-116/1a23a7a/semi-sync.log) |
| service-per-pod | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-116/1a23a7a/service-per-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-116/1a23a7a/scaling.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-116/1a23a7a/sidecars.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-116/1a23a7a/users.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-116/1a23a7a/limits.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-116/1a23a7a/demand-backup.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-116/1a23a7a/gr-init-deploy.log) |
| We run 12 out of 12|

commit: https://github.com/percona/percona-server-mysql-operator/pull/116/commits/1a23a7adcabbf968e9ec495804141b694f15af48
image: `perconalab/percona-server-mysql-operator:PR-116-1a23a7a`

**Comment by inelpandzic** [comment]
Hey @hors , could you re-check it pls? This is the result now we have:
- async replication
```
➜  k get perconaservermysql
NAME       REPLICATION   ENDPOINT                         STATE   ORCHESTRATOR   ROUTER   AGE
cluster1   async         cluster1-mysql-primary.default   ready   3                       7m49s
```

- group-replication
```
➜  k get perconaservermysql
NAME       REPLICATION         ENDPOINT                  STATE   ORCHESTRATOR   ROUTER   AGE
cluster1   group-replication   cluster1-router.default   ready                  3        5m9s
```

**Comment by inelpandzic** [comment]
> > Hey @hors , could you re-check it pls? This is the result now we have:
> > 
> > * async replication
> > 
> > ```
> > ➜  k get perconaservermysql
> > NAME       REPLICATION   ENDPOINT                         STATE   ORCHESTRATOR   ROUTER   AGE
> > cluster1   async         cluster1-mysql-primary.default   ready   3                       7m49s
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * group-replication
> > 
> > ```
> > ➜  k get perconaservermysql
> > NAME       REPLICATION         ENDPOINT                  STATE   ORCHESTRATOR   ROUTER   AGE
> > cluster1   group-replication   cluster1-router.default   ready                  3        5m9s
> > ```
> 
> @inelpandzic cool but what about MYSQL. Please see my comment above.

**Review by tplavcic — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**